tools/bin/check-duplicated-storages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+backup_storage_output=$(kubectl get backupstorage -n everest-system)
+backup_storage_names=($(echo "$backup_storage_output" | awk 'NR>1 {print $1}'))
+
+get_value() {
+  local map="$1"
+  local key="$2"
+  local value=$(echo "$map"| grep "$key" | cut -d"=" -f2)
+  echo $value
+}
+
+echo "Backup storages which share the same bucket, region, endpointURL:"
+found=0
+buckets=""
+
+for name in "${backup_storage_names[@]}"; do
+  storage=$(kubectl get backupstorage "$name" -n everest-system -o json)
+
+  bucket=$(echo "$storage" | jq -r '.spec.bucket')
+  endpointURL=$(echo "$storage" | jq -r '.spec.endpointURL')
+  region=$(echo "$storage" | jq -r '.spec.region')
+
+  key="$bucket$endpointURL$region"
+  storage_name=$(echo "$storage" | jq -r '.metadata.name')
```

Comment:
I'm trying to evaluate if it would be very bad to do 5 `kubectl get` commands using `-o jsonpath={.spec.bucket}` (adjusted for the other fields) so that we can remove the `jq` dependency...